adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -74,8 +82,34 @@ class FastqRecordConverter extends Serializable with Logging {
         .build())
   }
 
+  def parseMultiLine(element: String): Array[String] = {
+    val fastqRegex = "\\n(?=\\+)"
+
+    val splitOne = element.split("\\n", 2)
```

Comment:
What is all the splitting necessary here for? If you could add a few inline comments, that would help to clarify.